commands/mining.go

```diff
@@ -48,6 +50,14 @@ var miningStartCmd = &cmds.Command{
 	Encoders: stringEncoderMap,
 }
 
+var miningStatusCmd = &cmds.Command{
+	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
+		isMining := GetBlockAPI(env).MiningStatus()
+		return re.Emit(strconv.FormatBool(isMining))
+	},
+	Encoders: stringEncoderMap,
+}
```

Comment:
It would be great to have a `Type` on this so we can parse the response when requesting JSON encoded output. The `miningOnceCmd` command above provides a simple example.

```
type MiningStatusResult struct {
    Active bool
}
```

This will help not break things if we ever need to add additional information to the status command at a later time.

Reply:
@travisperson Thanks for your help, I've addressed it.